_topic_map.yml

```diff
@@ -763,6 +726,42 @@ Topics:
     File: monitoring-and-cluster-logging-operator-component-certificates
   - Name: Control plane certificates
     File: control-plane-certificates
+- Name: Compliance
+  Dir: compliance_operator
+  Topics:
+  - Name: Installing the Compliance Operator
+    File: compliance-operator-installation
+  - Name: Compliance Operator scans
+    File: compliance-scans
+  - Name: Understanding the Compliance Operator
+    File: compliance-operator-understanding
+  - Name: Managing the Compliance Operator
+    File: compliance-operator-manage
+  - Name: Tailoring the Compliance Operator
+    File: compliance-operator-tailor
+  - Name: Retrieving Compliance Operator raw results
+    File: compliance-operator-raw-results
+  - Name: Managing Compliance Operator remediation
+    File: compliance-operator-remediation
+  - Name: Performing advanced Compliance Operator tasks
+    File: compliance-operator-advanced
+  - Name: Troubleshooting the Compliance Operator
+    File: compliance-operator-troubleshooting
+  - Name: Using the oc-compliance plug-in
```

Comment:
@ahardin-rh Can you please take a look at the change I made here?

I moved the oc-compliance plug-in assembly in with the compliance operator docs (since it's directly related to it), and then renamed the overall folder to "Compliance".

Reply:
Excellent, thank you!